### PR TITLE
Dev-newPostsData

### PR DESCRIPTION
**Pull request by DaGigabyte**
listen for and load new post into MobX store
auto display posts tentatively using `updateHomePageDataWithNewPosts` (may change to a notification allowing the user to see the new post upon clicking the notification instead)

### CODE REVIEW

File: src/services/firePinsSource.js
[the remark this answers is not in the document]
Comment (DaGigabyte) [reply src/services/firePinsSource.js:1]: What does `source_id` store?

[the remark this answers is not in the document]
Comment (ziamohseni) [reply src/services/firePinsSource.js:1]: > What does `source_id` store?

"source_id" is storing the "id" from source API, in our example TMDB.

[the remark this answers is not in the document]
Comment (ziamohseni) [reply src/services/firePinsSource.js:1]: > Do you want to store the following `id` as `source_id`? image

yes.
And you see the "genre_ids" too.

[the remark this answers is not in the document]
Comment (DaGigabyte) [reply src/services/firePinsSource.js:1]: Please check the following PR for createPostEditor related changes
https://github.com/DaGigabyte/DH2642_Gp45/pull/17

**Pull request merged**